### PR TITLE
Add missing Storefront operation to pause an order

### PR DESCRIPTION
**Pull request by ashkarpetin**
## Summary
Add missing Storefront operation to pause an order
Rename GetEventRuleHistoryVersion operation to GetEventRuleHistory for consistency with API

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/requestBodies/storefront/PostOrderPause.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        description:
+          description: Description of the pause reason in free form.
+          type: string
+          maxLength: 255
+          nullable: true
+        effectiveTime:
+          description: |-
+            Date when the service period pauses.
+
+            This time should be later than the current time.
```
Comment (lexerom) [inline openapi/components/requestBodies/storefront/PostOrderPause.yaml:15]:
```suggestion
```
Avoid using 'should'. Not sure if we need to delete it, but without it meaning stays the same.

File: openapi/components/requestBodies/storefront/PostOrderPause.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      type: object
+      properties:
+        description:
+          description: Description of the pause reason in free form.
+          type: string
+          maxLength: 255
+          nullable: true
+        effectiveTime:
+          description: |-
+            Date when the service period pauses.
```
Comment (lexerom) [inline openapi/components/requestBodies/storefront/PostOrderPause.yaml:13]:
```suggestion
            Date and time when the service period pauses.
```

Comment (ashkarpetin) [reply openapi/components/requestBodies/storefront/PostOrderPause.yaml:13]: I copied this from SubscriptionPause schema

https://github.com/Rebilly/api-definitions/blob/f5710175ff73cb281089abc8e2269f9773d68d09/openapi/components/schemas/Subscription/SubscriptionPause.yaml#L40

File: openapi/components/requestBodies/storefront/PostOrderPause.yaml (hunk 1 above)
Comment (ashkarpetin) [reply openapi/components/requestBodies/storefront/PostOrderPause.yaml:15]: @noon-dawg  If there is mistake we need to fix in both places

Comment (colinbird) [reply openapi/components/requestBodies/storefront/PostOrderPause.yaml:15]: > This time must be later than the current time.

?

Comment (ashkarpetin) [reply openapi/components/requestBodies/storefront/PostOrderPause.yaml]: I made the changes.

**Review by colinbird — APPROVED** [review]

**Review by lexerom — APPROVED** [review]

File: openapi/components/requestBodies/storefront/PostOrderPause.yaml (hunk 1 above)
Comment (noon-dawg) [reply openapi/components/requestBodies/storefront/PostOrderPause.yaml]: We should avoid using 'should' when something is required. 'Should' may seem optional and may also sound passive.

I must have missed that usage in `[api-definitions/openapi/components/schemas/Subscription/SubscriptionPause.yaml](https://github.com/Rebilly/api-definitions/blob/f5710175ff73cb281089abc8e2269f9773d68d09/openapi/components/schemas/Subscription/SubscriptionPause.yaml#L40). I'll update it now. Sorry about that

**Pull request merged**